src/benchmarklib/jcch/jcch_table_generator.cpp

```diff
@@ -23,6 +23,10 @@ JCCHTableGenerator::JCCHTableGenerator(const std::string& dbgen_path, const std:
 std::unordered_map<std::string, BenchmarkTableInfo> JCCHTableGenerator::generate() {
   const auto tables_path = _path + "/tables/";
 
+  // NOLINTBEGIN(concurrency-mt-unsafe)
+  // clang-tidy complains that system() is not thread-safe. We ignore this warning as we expect that users will not call
+  // the JCCH table generator in parallel.
```

Comment:
Asserting means we need to somewhat keep track of concurrent calling. Since the methods using `system()` will probably throw anyway (e.g., by adding the same table twice) when called concurrently, I wouldn't add a safe guard. You cannot trigger the case without misusing the system.